Makefile

```diff
@@ -306,7 +306,7 @@ docker-%:
 	image=$* ;\
 	$(MKDIR) -p $(DIST_DIR)/$${image/:} ;\
 	$(DOCKER) build --network=host \
-                    --build-arg IMAGESPEC=$* \
+                    --build-arg IMAGESPEC=$${image//_//} \
```

Comment:
How do you invoke this target?

Reply:
```console
$ make docker-opensuse_leap:15.1 TAG=1
```
... pretty much the same way as you would for a centos, just that the name is now `opensuse_leap`.

The catch here is that there is more than one openSUSE distribution: [LEAP](https://hub.docker.com/r/opensuse/leap) and [Tumbleweed](https://hub.docker.com/r/opensuse/tumbleweed), which are both in the `opensuse` namespace. It would be preferable to do this directly in `Dockerfile.opensuse_leap` instead, but that doesn't work afaik.

Reply:
instead of an `_`, we could also use a `-`, which is what openSUSE itself is using in `/etc/os-release`